src/main/java/com/github/dockerjava/api/command/BuildImageCmd.java

```diff
@@ -87,6 +88,9 @@
     @CheckForNull
     public String getCpusetcpus();
 
+    @CheckForNull
+    public Map<String, String> getBuildArgs();
```

Comment:
That's docker version, not docker API https://docs.docker.com/engine/reference/api/docker_remote_api/